jwst/guider_cds/guider_cds_step.py

```diff
@@ -21,7 +23,29 @@ class GuiderCdsStep (Step):
 
     def process(self, input):
         with datamodels.GuiderRawModel(input) as input_model:
-            out_model = guider_cds.guider_cds(input_model)
+
+            # Get the gain reference file
+            gain_model = None   # will overwrite if file can be retrieved
+            try:
+                gain_filename = self.get_reference_file(input_model, 'gain')
+                gain_model = datamodels.GainModel(gain_filename)
+                self.log.info('Using GAIN reference file: %s', gain_filename)
```

Comment:
@hbushouse Does this need to be an `info` or a `debug` message? What do we usually do in other steps?

Reply:
It's usually an info level message, so this is OK.